src/NCalc.Core/Parser/LogicalExpressionParser.cs

```diff
@@ -160,7 +160,7 @@ static LogicalExpressionParser()
 
         // ("[" | "{") identifier ("]" | "}")
         var identifierExpression = openBrace.Or(openCurlyBrace)
-            .SkipAnd(AnyCharBefore(closeBrace.Or(closeCurlyBrace), consumeDelimiter: true))
+            .SkipAnd(AnyCharBefore(closeBrace.Or(closeCurlyBrace), consumeDelimiter: true, failOnEof: true).ElseError("Brace not closed."))
```

Comment:
Thank your for this PR!

Can you please rename "Brace" to "Braces", to match "Parenthesis not closed." error

Reply:
I don't think that's correct. "Parenthesis" is singular, while "parentheses" is plural.

Reply:
> I don't think that's correct. "Parenthesis" is singular, while "parentheses" is plural.

No English speak here lol, you change this typo if you want